locales/en.json

```diff
@@ -100,6 +104,23 @@
   "roadmap.state.unstarted": "Planned",
   "roadmap.suggest_a_feature": "Suggest a feature",
   "roadmap.title": "Roadmap",
+  "server.category.academia": "Academia",
+  "server.category.activism": "Activism",
+  "server.category.adult_content": "Adult content",
+  "server.category.art": "Art",
+  "server.category.books": "Books",
+  "server.category.food": "Food",
+  "server.category.furry": "Furry",
+  "server.category.gaming": "Gaming",
+  "server.category.general": "General",
+  "server.category.humor": "Humour",
+  "server.category.journalism": "Journalism",
+  "server.category.lgbt": "LGBTQ+",
+  "server.category.music": "Music",
+  "server.category.regional": "Regional",
+  "server.category.religion": "Religion",
+  "server.category.sports": "Sports",
+  "server.category.technology": "Technology",
```

Comment:
These were added when I ran the `extract` command. Not sure if they should be removed.

Reply:
This is because you added `data` to the I18n extract command. It was missing before (maybe due to a change I made to our i18n system some months ago).

Reply:
ah, I see. Since I'm going to remove the remove the localized bios, I'll just remove these loc changes as well since they're not related to the scope of this PR

Reply:
Feel free to open a PR with those changes, we probably need to make this work again.